elasticsearch/run.sh

```diff
@@ -25,20 +34,20 @@ if [ -z $(curl -s -f 'http://localhost:9200/searchguard/ac/ac') ]; then
       },
       {
         "__Comment__": "This is so that fluentd can only write",
-        "users": ["fluentd"],
+        "users": ["system.logging.fluentd"],
         "filters_bypass": [],
         "filters_execute": ["actionrequestfilter.fluentd"]
       },
       {
         "__Comment__": "This is so that Kibana can do anything in the .kibana index",
-        "users": ["kibana"],
-        "indices": [".kibana"],
+        "users": ["system.logging.kibana"],
+        "indices": [".kibana*"],
```

Comment:
Nit but this should probably be .kibana.* so that we don't accidentally allow access to something like .kibanaforsomeoneelse.* (not sure if that's an actual use case)